readme.md

```diff
@@ -16,107 +16,16 @@
 
```

Comment:
should add a title here `# TechTestApp`

Reply:
Also, I think bring `doc/readme.md` back into here. Otherwise, put a couple of links to `doc/readme.md` and `ASSESSMENT.md` in here somewhere

Reply:
I wanted to split the readme for the app from the test, why I went with the doc/readme.md. It's already linked at line 23

There is already a title in the ./readme.md file, `# Vibrato Tech Test` or do you think it should be  changed?